quirks/password-rules.json

```diff
@@ -202,6 +202,6 @@
         "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; required: digit;"
     },
     "signin.ea.com": {
-     "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;" 
+        "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;" 
```

Comment:
Thanks for fixing this! It looks like EA allows a few special characters that aren’t required, like `!@#-^&*=+;:`.

Can you augment these rules with something like `allowed: [!@#-^&*=+;:]`, and re-validate the rule is valid using the developer tool? Then I think we’ll be good to go here!